README.md

```diff
@@ -1,24 +1,24 @@
 # CaptainFact dev data
 
-This repo is intended to share some data we use in development environments
+The intention of this repo is to share some data used in development environments
 or that we got from third parties.
 
-It also contains Dockerfiles to build a postgres database with everything needed
+It also contains Dockerfiles to build a PostgreSQL database with everything needed
 to get started quickly.
 
 ## Structure
 
 ### Wikidata
 
-Contains all the data extracted from wikidata via wikidata. The `.sparql`
+Contains all the data extracted from Wikidata via wikidata. The `.sparql`
```

Comment:
```suggestion
Contains all the data extracted from Wikidata via the [Wikidata Query Service](https://query.wikidata.org/) API. The `.sparql`
```

Reply:
I don't have permission to edit PR, will merge and update directly

Reply:
![image](https://user-images.githubusercontent.com/1556356/60383393-479d0c00-9a70-11e9-8599-33e4dc3dc566.png)